.github/workflows/mypy-type-check.yml

```
@@ -6,7 +6,7 @@ on:
   push:
     branches: [ develop, pre-release, master, main ]
   pull_request:
-    branches: [ develop, pre-release, master, main ]
+    branches: [ develop, pre-release, master, main, enhance-torch-compile ]
```

```suggestion
    branches: [ develop, pre-release, master, main]
```

Please delete this now as this PR is ready.